tx.go

```diff
@@ -106,3 +113,37 @@ func ExtractMsgFromSum(sum interface{}) (Msg, error) {
 	}
 	return res, nil
 }
+
+// TxLoad extracts the message represented by given transaction into given
+// destination. Before retutning message validation method is called.
+func TxLoad(tx Tx, destination interface{}) error {
+	msg, err := tx.GetMsg()
+	if err != nil {
+		return errors.Wrap(err, "cannot get transaction message")
+	}
+
+	if err := msg.Validate(); err != nil {
```

Comment:
If we want to be paranoid, we should also check for `msg == nil` before this point

Reply:
Done in https://github.com/iov-one/weave/pull/467/commits/77a9c1a1aff660418f1b9d567580452410d47405